naga/src/back/mod.rs

```diff
@@ -215,6 +215,8 @@ impl FunctionCtx<'_> {
                     external_texture_key,
                 )
             }
+            // On rust 1.91 this will suddenly trigger clippy::panic.
+            #[allow(clippy::panic)]
```

Comment:
should prefer `expect`. `allow`s tend to accumulate

edit: is this version dependent like some of the ones below?